kivy/tests/common.py

```diff
@@ -427,18 +427,20 @@ def __init__(self, x, y):
         '''Create a MotionEvent instance with X and Y of the first
         position a touch is at.
         '''
-
         from kivy.base import EventLoop
         self.eventloop = EventLoop
         win = EventLoop.window
-
         super(UnitTestTouch, self).__init__(
             # device, (tuio) id, args
             self.__class__.__name__, 99, {
                 "x": x / (win.width - 1.0),
                 "y": y / (win.height - 1.0),
-            }
+            },
+            is_touch=True,
+            type_id='touch'
         )
+        # set profile to accept x, y and pos properties
+        self.profile = ['pos']
```

Comment:
This worries me a little, because normally you'd overwrite `depack`, but now, when you inherit, you'll have to make sure to set the profile after calling super in `__init__`. But it should be ok.

Reply:
When `profile` does not change in `depack` then we can just assign it in `__init__`. Depending on a use case another subclass can still change it in the `depack` method.